<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageKernel</class>
 <widget class="QWidget" name="PageKernel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="kernelsTab">
      <attribute name="title">
       <string>Kernels</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListView" name="kernelListView">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="notificationsTab">
      <attribute name="title">
       <string>Notifications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="notificationsGroupBox">
         <property name="title">
          <string>Notifications</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="unsupportedKernelCheckBox">
            <property name="text">
             <string>Notify for installed unsupported kernels</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="unsupportedKernelHBoxLayout">
            <item>
             <spacer name="unsupportedKernelBoxHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="unsupportedKernelRunningCheckBox">
              <property name="text">
               <string>Only if running the unsupported kernel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="newKernelCheckBox">
            <property name="text">
             <string>Notify when new kernels are available</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="newKernelLtsHBoxLayout">
            <item>
             <spacer name="newKernelLtsHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="newKernelLtsCheckBox">
              <property name="text">
               <string>Only LTS kernels</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="newKernelRecommendedHBoxLayout">
            <item>
             <spacer name="newKernelRecommendedHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="newKernelRecommendedCheckBox">
              <property name="text">
               <string>Only recommended kernels</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="notificationsVerticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
